packages/editor/src/store/private-selectors.js

```diff
@@ -93,37 +97,6 @@ const CARD_ICONS = {
 	post: verse,
 };
 
-export const getPostIcon = createRegistrySelector(
```

Comment:
I kept this selector but made it a bit more easy to work with (receives postId instead of options)